desafio1.json

```
@@ -0,0 +1,16 @@
+{
+  "coluna_usuario": "usuario_id",
```

```suggestion
  "coluna_usuario": "usuario",
```
Tem que ser o nome da coluna que guarda o nome do usuario, não o ID dele !
:eyes: